### DISCUSSION TITLE
Cannot resubmit better student identity card photo after verification rejection

### DISCUSSION THREAD

**Discussion by MWronski12** [community/community#120751] [GitHub Education]
### Hi everyone,

Hi, 

I uploaded a photo of my student card in order to get verified for the student developer pack, but it was rejected due to not enough details available on the photo. 

Now I am trying to submit a better quality photo of both sides of my identity card, so my personal data and due date is clearly visible, but online form does not accept it. After selecting the photo and clicking submit button I get redirected back to the clean form view without any processing taking action.

Please let me know how this can be fixed!

**Comment by MindOfBear** [comment]
Hello @MWronski12,

Before doing something, try another browser. If the issue still persist, create a ticket and you will get help.
Three days ago I applied for GitHub Student aswell, after the application got canceled, i submited another photo with the back of my identity card but I havent completed all the forms and i tought i encountered a error, so, also check if you have any empty form.

I hope your issue will be solved.

Have a great day,
Ovidiu.
